rocky/reports/runner/report_runner.py

```diff
@@ -206,8 +206,10 @@ def create_asset_reports(
                 manual_mime_types={"openkat/report"},
             )
 
+            input_ooi = reference if not hasattr(reference, "human_readable") else reference.human_readable
```

Comment:
It doesn't matter much, but it's a bit easier to read the statement if it's the other way around
```suggestion
            input_ooi = reference.human_readable if hasattr(reference, "human_readable") else reference
```
I'll leave it up to you